### ISSUE TITLE
Preempt paragraphs between tables

### ISSUE THREAD

**Issue by opoudjis** [riboseinc/isodoc#69]
Examples in Word are being output as tables. When there are two tables next to each other in Word HTML, Word interpolates a carriage return between them. And because Word is interpolating it, Word puts its own style on it.

If the two tables are embedded in a list, e.g.

````
. Complete qualification (<<complete-qualification>>) is preferred over group qualification (<<group-qualification>>), which is preferred over individual qualification of time scale components (<<individual-qualification>>);
+
--
[example]
'2015-02-28?' is preferred over '2015-02?-?28' because complete qualification is preferred.

[example]
'2015-02?-28' is preferred over '?2015-?02-28' because group qualification is preferred.
--
````

then the paragraph that Word interpolates inherits whatever styling a paragraph would get at that point. For the example above, that means:

````
<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;mso-list:l0 level1 lfo1'><a
name="_cc38bed6-9082-4c4f-945b-ee5386497f5e"
id="_cc38bed6-9082-4c4f-945b-ee5386497f5e"></a><![if !supportLists]><span
lang=EN-GB><span style='mso-list:Ignore'>c)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Redundant qualifications should
be removed;<o:p></o:p></span></p>
````

Since we're in a list, and adding a paragraph break would generate a new list item, the second example is being rendered with a `b)` before it, as a new list item.

To prevent that, we should insert our own default-styled paragraphs between any two tables in Word output, to prevent them picking up list styling.

Eventually we may also want to shrink the size of these paragraphs: consecutive examples currently have distractingly large space between them.

**Comment by opoudjis** [comment]
The paragraph is still being converted to a list item. The only way to prevent that is to stop using HTML markup for lists at all, and use the fully native Word HTML strategy of treating all list items as paragraphs with an mso-list attribute. I already have had to add mso-list attributes to `<li>`, to force correct list numbering; so I have modified html2doc to convert `<ul><li mso-list=...><p>` and `<ol><li mso-list=...><p>` to `<p mso-list=...>`. Once that is done, any paragraphs within the list item (in reality, any paragraphs now following the list item paragraph) no longer get a list item number/bullet.

To get indentation happening correctly, I also have to switch the paragraph styles from MsoNormal to the native MsoListParagraphCxSpFirst, MsoListParagraphCxSpMiddle, MsoListParagraphCxSpLast.

All Word stylesheets need to be updated to use the native paragraph styles.

**Comment by opoudjis** [comment]
.... Actually it looks like they don't: the indentation solution I had to put in to address the implicit indentation of ol and ul goes away when I don't use ol and ul to begin with.

There is a new issue now. There is no longer a new list item for the second table under a list item; but that second table is no longer indented under the same level as the first, either. Will see if a div wrapper addresses that.

**Comment by opoudjis** [comment]
Will make that a new issue.
